recipes/cctz/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class CCTZConan(ConanFile):
+    name = "cctz"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/cctz"
+    description = "C++ library for translating between absolute and civil times"
+    topics = ("conan", "cctz", "time", "timezones")
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "build_tools" : [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False, 
+        "build_tools": False
```

Comment:
What are the implications of setting the default to true?

Reply:
It builds an executable. Should I remove this for now or add it to PATH conditionally?

Reply:
Is this executable something people normally use? Does it have any use as a build require for other Conan recipes?

Reply:
I think we can put both together in this package, is not a usual executable. Also, you can put it to PATH conditionally

Reply:
Done. Gets build, packaged, tested and add to PATH conditionally.

Good catch by the way @danimtb, this is a bug in the Bincrafters' version. We build the `time_tool` executable there when `build_tools=True` but then don't actually add it to the package.....